.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+  },
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  ignorePatterns: ['src/main/webapp/*'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+  ],
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: ['./tsconfig.json'],
+  },
+  rules: {
+    'no-extra-semi': 'off',
+    'prefer-const': 'warn',
+    '@typescript-eslint/restrict-template-expressions': [
+      'warn',
+      { allowBoolean: true },
+    ],
+    '@typescript-eslint/unbound-method': 'warn',
+    '@typescript-eslint/no-inferrable-types': 'off',
+    '@typescript-eslint/ban-ts-comment': 'warn',
+    '@typescript-eslint/no-empty-interface': 'warn',
+    '@typescript-eslint/no-floating-promises': 'error',
```

Comment:
For the most part, these rules are a direct subset of rules in [synapse-web-monorepo](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/main/.eslintrc.cjs). However, `@typescript-eslint/no-floating-promises` is set to error to catch issues in Playwright tests, as recommended in their [best practices](https://playwright.dev/docs/best-practices#lint-your-tests) guide.

Reply:
Do you think we should create a ticket to enable this rule in synapse-web-monorepo? I think it makes sense since we're writing tests in essentially the same way

Reply:
I think it would be good to create the ticket since we can use the `void` keyword to be explicit about the places where we don't want to handle the promise. Do you mind creating it?

Reply:
Sure! https://sagebionetworks.jira.com/browse/PORTALS-2894